Classes/View/PdfView.php

```diff
@@ -161,6 +161,7 @@ protected function getPdfObject()
         /* @var $pdf \mPDF */
         $pdf = $this->objectManager->get('mPDF', '', $pageFormat . '-' . $pageOrientation);
         $pdf->SetMargins($leftMargin, $rightMargin, $topMargin);
+        $pdf->_setPageSize($pageFormat, $pageOrientation);
```

Comment:
`_setPageSize` is apparently intended to be private (by underscore convention). I'd suggest passing the orientation into the `mPDF` constructor, instead.